### PR TITLE
feat(autocomplete): append classNames on template after sources

### PR DESCRIPTION
**Pull request by s-pace**
refactor: add story as playground
feat(dropdown): inject custom classNames to rendered sources
docs(autocomplete_option): add story for inject custom classNames after sources

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -20,6 +20,36 @@ export interface Environment {
   open: Window['open'];
 }
 
+/**
+ * Custom ClassNames appended to the template after the source.
```
Comment (francoischalifour) [inline src/types.ts]:
```suggestion
 * Custom class names appended to the template after the source.
```

File: src/types.ts [hunk 2]
```diff
@@ -116,6 +146,10 @@ export interface PublicAutocompleteSource {
    * Called when an item is selected.
    */
   onSelect?: (options: ItemEventHandlerOptions) => void;
+  /**
+   * Classes applied to the template after the source
```
Comment (francoischalifour) [inline src/types.ts]:
```suggestion
   * CSS classes applied to the template of the source
```

**Review by francoischalifour — APPROVED** [review]

**Comment by s-pace** [comment]
🚀

**Comment by s-pace** [comment]
🥇

**Pull request merged**